### DISCUSSION TITLE
index.html not found after cargo install

### DISCUSSION THREAD

**Discussion by grayfallstown** [tauri-apps/tauri#5995] [Q&A] [answered]
Is there a way to publish a tauri app as crate on crates.io?
When I publish it and then install the binary via cargo install, the binary starts, opens a webview and all I see is `ERR_CONNECTION_REFUSED`.
Tauri does not find my index.html, which is a static file in `./public`.
cargo run and cargo run --release work fine, just the publishing seems to break my tauri app.
`devDir` und `distDir` are both '`./public'`.

**Comment by FabianLars** [comment, marked as answer]
Looks like you fixed it? At least it seems to work on Windows. One word of warning though, `cargo install` does not install the system dependencies, most importantly the webview, so unless your users installed them by some other means the app will crash. (not a problem on macos where a webview is always present)
